maskerlogger/ahocorasick_regex_match.py

```diff
@@ -1,4 +1,4 @@
-import tomllib
+import tomli as toml
```

Comment:
Not familiar with `tomli`. I love `tomlkit`
This is only preference and obviously not a blocker